lib/intercom_stats/intercom/worker.ex

```diff
@@ -175,25 +168,47 @@ defmodule IntercomStats.Intercom.Worker do
     end
   end
 
-  defp calculate_closing_time(%{"created_at" => created}, close_timestamp, snooze_time) do
-    close_timestamp - created - snooze_time
-  end
-
-  defp determine_snooze_time(%{"conversation_parts" => %{"conversation_parts" => parts}}) do
+  defp determine_open_time(%{
```

Comment:
I know I did this, but lets use function names without words like `determine_..` or `calculate_..` prefixes. Just `open_time` would be sufficient